### DISCUSSION TITLE
Restrict outbound emails to specific domain.

### DISCUSSION THREAD

**Discussion by jaydeepnp** [postalserver/postal#2621] [Help with using Postal]
Hi Postal Team,

I have setup of 2 postal SMTP Server with IP pool , behind load balancer to deliver customer emails from  application.

Setup is working fine and  postal is amazing software.

I want to restrict delivery of outgoing  emails to specific domain  and  no try for delivery for that domain. Is there any way I can achieve this requirement.

Regards,
Jaydeep

**Comment by willpower232** [comment]
Unfortunately Postal is very happy to send all email it receives (or hold all email if specifically told to) so if you need this feature, you should probably implement it in whatever you're using to send mails via Postal.

**Reply by jaydeepnp** [reply]
Will,

Thank you for your reply, It is not completely clear for me. I understand that Postal sends all email it receives from its clients. For this feature i need to check at client (my Application) end. this part I got it. Thanks.

You mentioned  that we can specifically hold emails , Is it possible to hold all emails for specific email address or domain  for example abc@prod.xyz.com or @prod.xyz.com

Regards,
Jaydeep

**Reply by willpower232** [reply]
Sorry for the confusion, I should have added more emphasis but basically Postal will send everything you give it or nothing you give it. This is controlled by the following two settings and cannot currently be made more domain-specific.

![image](https://github.com/postalserver/postal/assets/1619102/dc8a394c-cc6d-4b31-af56-7164ac3f21a9)

![image](https://github.com/postalserver/postal/assets/1619102/0b59198c-0b38-449a-b24d-246322b93f0f)

If you need to apply limits then you should do that in whatever program you're using to pass messages to Postal.
